### ISSUE TITLE
Application error when trying to update content

### ISSUE THREAD

**Issue by bellangerq** [cobyism/ghost-on-heroku#124]
Hello there,

I tried to host my Ghost blog on Heroku. The installation worked fine but when I wanted to update the source code (*adding a theme in `content/themes/`*), my app crashed.

I follow the instructions from the [README](https://github.com/mars/ghost-on-heroku/blob/version-1.x/README.md#updating-source-code):

> Optionally after deployment, to push Ghost upgrades or work with source code, clone this repo (or a fork) and connect it with the Heroku app:

```
git clone https://github.com/cobyism/ghost-on-heroku
cd ghost-on-heroku

heroku git:remote -a YOURAPPNAME
heroku info
```
> Then you can push commits to the Heroku app, triggering new deployments:
```
git add .
git commit -m "Important changes"
git push heroku master
```

After pushing code and restarting my dynos, my app just crashed with this error in the Heroku logs:
```
at=error code=H10 desc="App crashed" method=GET path="/" host=neige-theme.herokuapp.com request_id=953cd8bb-5dc6-4e28-91fc-90e5eec9e230 fwd="86.196.249.249" dyno= connect= service= status=503 bytes= protocol=https
```

I'm quite new to Heroku so I wanted to know if I did something wrong or if it's a current issue. Thanks for helping and for creating such simple process to deploy Ghost blog :)

**Comment by ozzyonfire** [comment]
Any news on this issue? I attempted the same thing. Looking at the logs in heroku I see the error:
`Unhandled rejection DatabaseIsNotOkError: Migrations are missing. Please run knex-migrator migrate.`

I get this even when removing the custom theme from the themes folder.

**Comment by ozzyonfire** [comment]
After looking through some of the closed issues, I figured out what I had to do.
I added a couple scripts to the package.json that makes building and deploying from Git much easier. Now I added these two lines to the 'scripts'.

`"heroku-postbuild": "bin/create-config && bin/init-deployment", // this allows you to deploy from git and have it build successfully`
`"prestart": "knex-migrator migrate --mgpath node_modules/ghost && bin/wait-for-db" // this runs the migrator and then waits for the db to connect just before heroku starts up your app`
